src/calculateRentalCost.js

```diff
@@ -25,10 +25,21 @@
  * @param {number} days
  *
  * @return {number}
+ *
  */
 
 function calculateRentalCost(days) {
   // write code here
+  const costPerDay = 40;
+  let totalCost = costPerDay * days;
```

Comment:
```suggestion
  const totalCost = costPerDay * days;
```
![image](https://user-images.githubusercontent.com/96409563/229736922-661f0a6e-6b37-4e9b-9ff0-4df2682e1211.png)